src/js/plugins/tts/FestivalTTSEngine.js

```diff
@@ -120,4 +104,23 @@ export default class FestivalTTSEngine extends AbstractTTSEngine {
                   + encodeURIComponent(dataString)
                   + '&format=.'+this.audioFormat;
     }
-}
\ No newline at end of file
+
+    /**
+     * @private
+     * Security restrictions require playback to be triggered
+     * by a user click/touch. This intention gets lost in the async calls
+     * on iOS, but, for some reason, if we start the audio here, it works.
+     * See https://stackoverflow.com/questions/12206631/html5-audio-cant-play-through-javascript-unless-triggered-manually-once
+     * @return {PromiseLike}
+     */
+    iOSCaptureUserIntentHack() {
+        let sound = soundManager.createSound({
+            // We can't use a whitespace string; it causes an infinite loop (???)
+            // Let's choose a short sound, cause it'll still wait for it to finish
+            url: this.getSoundUrl('t'),
+            volume: 0
```

Comment:
SUGGESTION: For AirBNB style, add a comma.

Reply:
Done